src/modules/form/util/formUtil.tsx

```diff
@@ -1268,18 +1269,16 @@ export const transformSubmitValues = ({
 };
 
 // record could be an Assessment
-const getMappedValue = (record: any, mapping: FieldMapping) => {
+const getMappedValue = (
+  record: any,
+  mapping: FieldMapping,
+  raiseOnMissing: boolean
```

Comment:
The passing around of `sentryOnMissing` and `raiseOnMissing` feels a little too prescriptive. Maybe we could make it more flexible by just accepting a callback method for handling missing fields? For example:

* `createInitialValuesFromRecord({ handleMissingField = sendToSentry })`  defaults to passing `sendToSentry` for handling missing attributes
* `createInitialValuesFromRecord({ ..., handleMissingField: () => ()})` in the one place in AssessmentForm where we want to override this behavior, we can pass a void function

What do you think?

Reply:
Thanks, that seems reasonable to me!

Reply:
... but there is some problem with passing `sendToSentry` as a default param, storybook build chokes on it at the react docgen step, I think. https://github.com/greenriver/hmis-frontend/actions/runs/14174457600/job/39705834696#step:9:631. I'll keep digging.

Reply:
Hmm odd. I looked at this a bit. I wouldn't expect it to run docgen on that file since it's a util file. It doesn't need to be a `tsx` file since it doesn't contain any jsx. changing it to `ts` seems to fix it

Reply:
Cool, thank you